content/docs/guides/06_light_push_send_messages.md

```diff
@@ -76,22 +80,252 @@ The returned `Promise` will resolve once you are connected to a Waku peer.
 
 # Send messages
 
-You can now send a message using Waku Light Push.
-By default, it sends the messages to a single randomly selected light push peer.
-The peer is selected among the dApp's connected peers.
+To send a message, you need to use the `@waku/create` library to create a Waku Light Node and the `@waku/core` library to encode the message. 
+
+## Import the required libraries
+
+```ts
+import * as utils from "@waku/byte-utils";
+import * as wakuCreate from "@waku/create";
+import { createEncoder, waitForRemotePeer } from "@waku/core";
+```
+
+## Define the required constants
+
+```ts
+const MULTI_ADDR = "your Waku multi-address";
+const CONTENT_TOPIC = "your content topic";
+const PROTOCOLS = ["filter", "lightpush"];
+```
+
+## Create a Waku Light Node
+
+```ts
+const node = await wakuCreate.createLightNode({ defaultBootstrap: true });
+await node.start();
+await waitForRemotePeer(node, protocols);
+```
+
+## Create an encoder for the message
+
+```ts
+const Encoder = createEncoder(CONTENT_TOPIC);
+```
+
+## Define the structure of the message
+
+```ts
+const ChatMessage = new protobuf.Type("ChatMessage")
+  .add(new protobuf.Field("timestamp", 1, "uint64"))
+  .add(new protobuf.Field("nick", 2, "string"))
+  .add(new protobuf.Field("text", 3, "bytes"));
+```
+
+## Create a message object and encode it 
+
+```ts
+const message = ChatMessage.create({
+  nick: "your nick",
+  timestamp: Date.now(),
+  text: utils.utf8ToBytes("your message"),
+});
+const protoMessage = ChatMessage.encode(message).finish();
+```
+
+## Push the encoded message to the network
+
+```ts
+await node.lightPush.push(Encoder, {
+  payload: protoMessage,
+});
+```
+
+# Receive messages
+
+To receive messages, you need to subscribe to the Waku Filter, decode the incoming payload using a decoder specific to the content topic of the message, and then pass the decoded message to a callback function for processing.
+
+## Import the required libraries
+
+```ts
+import * as utils from "@waku/byte-utils";
+import * as wakuCreate from "@waku/create";
+import { createDecoder, waitForRemotePeer } from "@waku/core";
+```
+
+## Define the Waku Node connection parameters
+
+```ts
+const MULTI_ADDR = "/dns4/node-01.ac-cn-hongkong-c.wakuv2.test.statusim.net/tcp/443/wss/p2p/16Uiu2HAkvWiyFsgRhuJEb9JfjYxEkoHLgnUQmr1N5mKWnYjxYRVm";
+const CONTENT_TOPIC = "/toy-chat/2/huilong/proto";
+const PROTOCOLS = ["filter"];
+```
+
+## Create the Waku Node
+
+```ts
+const node = await wakuCreate.createLightNode({ defaultBootstrap: true });
```

Comment:
I would, probably, mention that `defaultBootstrap` uses default peer discovery mechanics.